src/components/TodoListItem.module.css

```diff
@@ -0,0 +1,119 @@
+/* change default button settings */
+
+/* navigation container */
+
+.todoListThreeColumn {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  flex-wrap: wrap;
+}
+
+.todoListColumn1 {
+  flex: 1;
+}
+
+.todoListColumn2 {
+  flex: 12;
+}
+
+.todoListColumn3 {
+  flex: 2;
+}
+
+.todoListItemButton {
+  letter-spacing: 2px;
+  border: none;
+  text-decoration: none;
+  width: 100px;
+  height: 40px;
+  margin-right: 20px;
+  border-radius: 25px;
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 0;
+  background: pink;
+  font-size: 9px;
+  color: black;
+  text-transform: uppercase;
+  font-family: 'helvetica';
+  font-weight: regular;
+  -webkit-transform: perspective(1px) translateZ(0);
+  transform: perspective(1px) translateZ(0);
+  box-shadow: 0 0 1px rgba(0, 0, 0, 0);
+}
+
+.todoListItemButton:hover {
+  background: rgb(209, 157, 166);
+}
+
+.todoListItemButton:hover {
```

Comment:
the hover animations are looking really nice! 

one thing still missing that makes the ux feel less responsive is there's no `::active` state on the button, which usually shows a different color etc for the instant that you're clicking

Reply:
haha thanks! I've added the active state on the button